curriculum/challenges/english/07-scientific-computing-with-python/learn-numerical-methods-by-finding-the-square-root-of-a-number-using-the-bisection-method/65ef1d924ff2e5c7031c6ce8.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 65ef1d924ff2e5c7031c6ce8
+title: Step 18
+challengeType: 20
+dashedName: step-18
+---
+
+# --description--
+
+Create an `else` condition to handle the case where the value of `root` is not `None` and found as a result of convergance. If it is not `None`, print the message `'The approximate square root of {square_target} is {root}.'`. Remember to format using an f-string.
```

Comment:
```suggestion
Create an `else` clause to handle the case where the value of `root` is not `None` and found as a result of convergence. If it is not `None`, print the message `'The square root of {square_target} is approximately {root}'`. Remember to format the message using an f-string.
```

Reply:
Maybe it's just me, but "found as a result of convergence" does not sound right.

Reply:
I modified the instruction a bit.